lib/auth/helpers.go

```diff
@@ -125,14 +136,7 @@ func NewTestAuthServer(cfg TestAuthServerConfig) (*TestAuthServer, error) {
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-
-	// set cluster name
-	clusterName, err := services.NewClusterName(services.ClusterNameSpecV2{
-		ClusterName: srv.ClusterName,
-	})
-	if err != nil {
-		return nil, trace.Wrap(err)
-	}
+	// set cluster name in the backend
 	err = srv.AuthServer.SetClusterName(clusterName)
```

Comment:
Now that auth server is initialized with `clusterName` (above), maybe it can be saved in `Init`?